IPython/core/history.py

```diff
@@ -8,13 +8,7 @@
 import datetime
 import os
 import re
-try:
-    import sqlite3
-except ImportError:
-    try:
-        from pysqlite2 import dbapi2 as sqlite3
-    except ImportError:
-        sqlite3 = None
+import sqlite3
```

Comment:
I think that not quite right; this make sqlite3 mandatory, before it would set slite3 to None, and still work.

Reply:
sqlite3 was [added in python 2.5](https://docs.python.org/2/library/sqlite3.html), so I assumed this import would never fail.

Reply:
That seem to have been added in 2012; (#2718), so my guess is some people have-had custom builds that do not have sqlite3  for whatever reason. 

It's been 6 years so let's see if this is still necessary. If we have complaints we can re-add.